sky/clouds/service_catalog/__init__.py

```diff
@@ -129,20 +129,20 @@ def get_region_zones_for_instance_type(
 
 
 def get_hourly_cost(instance_type: str,
+                    use_spot: bool,
```

Comment:
Is this from merge, or a new change? There seems quite a few places where orders aren't fixed:

```
» ag -w get_hourly_cost -A3

....<please check the rest>

sky/clouds/service_catalog/azure_catalog.py
36:def get_hourly_cost(instance_type: str,
37-                    region: Optional[str] = None,
38-                    zone: Optional[str] = None,
39-                    use_spot: bool = False) -> float:

sky/clouds/service_catalog/aws_catalog.py
85:def get_hourly_cost(instance_type: str,
86-                    region: Optional[str] = None,
87-                    zone: Optional[str] = None,
88-                    use_spot: bool = False) -> float:

sky/clouds/service_catalog/__init__.py
131:def get_hourly_cost(instance_type: str,
132-                    use_spot: bool,
133-                    region: Optional[str],
134-                    zone: Optional[str],

....<please check the rest>
```

Reply:
Thanks for letting me know. I think I omitted them by mistake. Fixed.